lib/config-util.js

```diff
@@ -219,6 +219,17 @@ Config.buildYargs = function (cwd) {
       type: 'boolean',
       global: false
     })
+    .option('clean', {
+      describe: 'should the .nyc_output folder be cleaned before executing tests',
+      default: true,
```

Comment:
I'm guessing this is because we want to just be explicit and start from a fresh state every time? There aren't really any perf benefits from keeping this folder around right?